docs/builds/guides/integration/configuration.md

```diff
@@ -133,6 +133,50 @@ ClassicEditor
 	} );
 ```
 
+### Extended format
+
+You can use the extended toolbar configuration format to access additional options:
+
+```js
+    toolbar: {
+        items: [ 'bold', 'italic', '|', 'undo', 'redo', '-', 'numberedList', 'bulletedList' ],
+        viewportTopOffset: 30,
+        shouldNotGroupWhenFull: true
+    }
+```
+
+ * **`items`** &ndash; An array of toolbar item names. The components (buttons, dropdowns, etc.) which can be used as toolbar items are defined in `editor.ui.componentFactory` and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`
```

Comment:
I know this is a straight copy from https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html#member-toolbar but mentioning `componentFactory` (which is a pretty low-level API) here in the guide for people who see the editor for the first time may not be the best idea. 

OTOH I don't know how we could avoid mentioning it. We don't have any guide for the component factory but it is used in the ["Creating a simple plugin" guide](https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/framework/guides/creating-simple-plugin.html) so maybe at least link there?

Reply:
Maybe listing all items somewhere in the guide docs and linking there instead of mentioning about componentFactory?

Reply:
This would be nice if we had a dedicated "toolbar guide" that was called for somewhere in the issues. List items, configuration and options. Maybe I should take it on next iteration.